core/src/trezor/strings.py

```diff
@@ -29,9 +29,14 @@ def format_amount_unit(amount: str, unit: str) -> str:
 
 
 def format_ordinal(number: int) -> str:
-    return str(number) + {1: "st", 2: "nd", 3: "rd"}.get(
-        4 if 10 <= number % 100 < 20 else number % 10, "th"
-    )
+    from trezor import translations
```

Comment:
IIUC, this function is currently used only in Caesar reset flow:
https://github.com/trezor/trezor-firmware/blob/963858c742da531d650d30a6bb4f931c76565b2d/core/src/trezor/ui/layouts/caesar/reset.py#L73-L101

Other layouts are using a bit different strings (`Select word {0} of {1}:` or `Select word #{0} from ...`) which IIUC don't require ordinal formatting.
Can we switch Caesar to use one of those?